.abf.yml

```diff
@@ -1,2 +1,2 @@
 sources:
-  cdrtools-3.01.tar.bz2: 62766aa9bfeec91b3c78da7c59b5c96bfbec84d9
+  cdrtools-3.02a01.tar.bz2: a7fccb3b16c7a4fc49d1926d72f20ebba290029a
```

Comment:
This should be updated when you abf put to the filestore. If you changed this manually, this will not work.